smpe/bld/AZWE002.htm

```diff
@@ -11,21 +11,21 @@ <H1>Program Directory for
   <BR> </H1>
 <H2>V0.0.1   </H2>
 <H2>  <H2>
-<H2>FMID AZWE001 </H2>
+<H2>FMID AZWE002 </H2>
```

Comment:
Note that we are not using the PD created here. The doc folks opted to create and maintain their own. IBM's version needs some IBM specific wording, and is therefore also maintained elsewhere.

Reply:
Well noted.